Classes/Domain/Model/Traits/FalFileTrait.php

```diff
@@ -60,6 +60,6 @@ public function getProperties(): ?array
 
     public function getPublicUrl(): string
     {
-        return $this->file->getPublicUrl();
+        return $this->file->getPublicUrl() ?? '';
```

Comment:
IMO it should be `public function getPublicUrl(): ?string` instead.

Reply:
This would also be possible, however, various parts would need to be adjusted because currently, a string is expected at many points. For example, in the File Model FalFile Model, and so on.

Reply:
Yes, I'd rather not change the public API for this issue.